tools/gce-quota-sync/gce-quota-sync.py

```diff
@@ -132,15 +132,15 @@ def _quota_to_series(project, region, quota):
     region: set in converted time series labels
     quota: quota object received from the GCE API
   """
-  labels = dict((k, str(v)) for k, v in quota.items() if k != 'usage')
+  labels = dict((k, str(v)) for k, v in quota.items())
```

Comment:
why was usage originally omitted / now included ?

Reply:
I have no idea. It was included in the examples as a feature and i definitely found the number to be useful in stackdriver.